coalib/settings/ConfigurationGathering.py

```diff
@@ -16,6 +16,13 @@
 from coalib.settings.Setting import Setting, path
 from string import Template
 
+COARC_OUTPUT = Template('$type \'$file\' $found!\n'
+                        'Here\'s what you can do:\n'
+                        '*add `--config .coarc --save` '
```

Comment:
this will be used when loading the `get_system_coafile` if it returns the file named `default_coafile`

Reply:
Also where is the `.coarc` located?